### PR TITLE
[stacked 8] Refactor Save/Notify to use interface DTO.

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

File: app/Contracts/PhotoCreate/PhotoDTO.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Contracts\PhotoCreate;
+
+use App\Models\Photo;
+
+interface PhotoDTO
```
Comment (d7415) [inline app/Contracts/PhotoCreate/PhotoDTO.php:7]: Should this file be in DTO instead of contracts?

**Review by d7415 — APPROVED** [review]

File: app/Contracts/PhotoCreate/PhotoDTO.php (hunk 1 above)
Comment (ildyria) [reply app/Contracts/PhotoCreate/PhotoDTO.php:7]: It is an interface, hence a contract.
I am open to another other name.

Comment (d7415) [reply app/Contracts/PhotoCreate/PhotoDTO.php:7]: Sure. Only because the others were in the DTO folder. If this makes more sense to you I'm happy as-is.

Comment (ildyria) [reply app/Contracts/PhotoCreate/PhotoDTO.php:7]: Could consider `PhotoDTOInterface` or `HasGetPhoto`. :)

Comment (d7415) [reply app/Contracts/PhotoCreate/PhotoDTO.php:7]: Could do - I think the current name is nice. Let's just leave it as-is.

**Pull request merged**